test/function/facility.cpp

```diff
@@ -53,11 +57,59 @@ namespace {
     int result;
 
     #define CHECK_FACILITY_ENUM( STRING, ENUM ) \
-      result = stumpless_get_facility_enum( #STRING ); \
+      result = stumpless_get_facility_enum( #STRING + 19 ); \
       EXPECT_EQ( result, ENUM );
     STUMPLESS_FOREACH_FACILITY( CHECK_FACILITY_ENUM )
   }
 
+  TEST( GetFacilityEnum, LowercaseFacility ) {
+    int result;
+
+    result = stumpless_get_facility_enum( "user" );
+    EXPECT_EQ( result, 8 );
```

Comment:
For each of these comparisons, use the enum symbol (e.g. `STUMPLESS_FACILITY_USER`) instead of the hardcoded number.

Reply:
Good point, done.